src/multiint.cpp

```diff
@@ -1260,6 +1260,13 @@ static bool recvVote(NETQUEUE queue)
 
 	debug(LOG_NET, "total votes: %d/%d", static_cast<int>(getVoteTotal()), static_cast<int>(NET_numHumanPlayers()));
 
+	// there is no "votes" that disallows map change so assume they are all allowing
+	if(newVote == 1) {
+		char msg[128] = {0};
+		snprintf(msg, 127, _("%s (%d) allowed map change. Total: %d/%d"), NetPlay.players[player].name, player, static_cast<int>(getVoteTotal()), static_cast<int>(NET_numHumanPlayers()));
+		sendRoomChatMessage(msg);
```

Comment:
The function `sendRoomChatMessage` could be replaced by `sendRoomSystemMessage`. "System" messages are displayed in yellow, without being prefixed by the player name and time. 

Alternatively, you could extend `RoomMessageType` with a new type of message, and then implement a new rendering for the message in `ChatBoxWidget::displayMessage(RoomMessage const &message)`.